src/main/java/com/boozallen/aissemble/UploadDependenciesMojo.java

```diff
@@ -0,0 +1,62 @@
+package com.boozallen.aissemble;
+
+import java.io.File;
+import java.nio.file.Paths;
+
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.project.MavenProject;
+
+import io.kokuwa.maven.helm.UploadMojo;
+import io.kokuwa.maven.helm.pojo.HelmRepository;
+import io.kokuwa.maven.helm.pojo.RepoType;
+
+@Mojo(name = "upload-dependencies")
+public class UploadDependenciesMojo extends UploadMojo {
+
+	@Parameter(property = "url", required = true)
+	private String url;
```

Comment:
A: If we use spaces in the pom, we should use spaces here as well.

Reply:
Fixed.